inst/stan/estimate_infections.stan

```diff
@@ -118,21 +118,18 @@ model {
 generated quantities {
   int imputed_reports[ot_h];
   vector[estimate_r > 0 ? 0: ot_h] gen_R;
-  real r[ot_h];
+  real r[ot_h - 1];
   vector[ot] log_lik;
-  if (estimate_r){
-    // estimate growth from estimated Rt
-    r = R_to_growth(R, gt_mean[1], gt_sd[1]);
-  }else{
+  if (estimate_r == 0){
     // sample generation time
     real gt_mean_sample = normal_rng(gt_mean_mean, gt_mean_sd);
     real gt_sd_sample = normal_rng(gt_sd_mean, gt_sd_sd);
     // calculate Rt using infections and generation time
     gen_R = calculate_Rt(infections, seeding_time, gt_mean_sample, gt_mean_sample,
                          max_gt, rt_half_window);
-    // estimate growth from calculated Rt
-    r = R_to_growth(gen_R, gt_mean_sample, gt_sd_sample);
```

Comment:
Now that `calculate_growth` and `R_to_growth` would have the same return value and vector length, you might keep both with a switch, if there's a value in keeping the `Rt` growth rate approximation (e.g., for diagnostics, not necessarily as a user option).